### PR TITLE
Update OCP cluster identifier instructions

### PR DESCRIPTION
**Pull request by boaz0**
Update the instructions at "Configure Cost Management Operator".

//cc @nlcwong @rvsia 

image

Thanks :)

### CODE REVIEW

File: packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js [hunk 1]
```diff
@@ -19,25 +18,17 @@ export const ConfigureCostOperator = () => (
...
-                    Enter the cluster identifier below:
+                    When configuration is complete, enter the cluster identifier below.<br />
+                    The cluster identifier can be found in the cluster&apos;s Help &gt; About screen
                 </TextListItem>
             </TextList>
         </TextContent>
     </TextContent>
 );
 
 export const ClusterIdentifierLabel = () => (
```
Comment (rvsia) [inline packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js:29]: you don’t need this anymore. Just set `label` and `isRequired` in the schema.

Comment (boaz0) [reply packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js]: Done

**Comment by nlcwong** [comment]
@boaz0 Text format nit: do not include a new line between "When configuration is complete,..." and "The cluster identifier ..." sentences. Those two sentences would look better if in the same paragraph.

**Comment by boaz0** [comment]
Done

**Comment by boaz0** [comment]
Thanks! :smile:

**Pull request merged**